docs/recipes/Post-To-Modules.md

```diff
@@ -0,0 +1,15 @@
+<!--ONE-DOCS-HIDE start-->
+[👈 Return to Overview](./README.md)
+<!--ONE-DOCS-HIDE end-->
+
+# POST To Module Routes
```

Comment:
Is this being fleshed out in a follow up story?

Reply:
This was just moved out as It does not make sense to be part of `Making an Api Call`. Can mark as WIP

Reply:
currently marked as a WIP

Reply:
We said that this section should be moved to the API docs IIRC